core/src/main/java/jenkins/util/AntClassLoader.java

```diff
@@ -772,6 +773,28 @@ private InputStream loadBaseResource(String name) {
         return parent == null ? super.getResourceAsStream(name) : parent.getResourceAsStream(name);
     }
 
+    private interface Convert<T> {
```

Comment:
I would prefer the interface to be called `Converter` because you use a converter to convert, so `Converter::convert`.. Also, it should be annotated with `@FunctionalInterface` to ease its comprehension (so clearly optional :wink:)